assets/src/scss/layout/_navbar.scss

```diff
@@ -290,13 +306,24 @@ $navbar-default-height: 60px;
 }
 
 .nav-item {
+  @include large-and-up {
+    margin-left: $sp-2x;
+  }
+}
+
+.nav-donate:hover .dropdown::after {
+  transform: rotate(-90deg);
+}
+
+.nav-item,
+.nav-donate {
   color: inherit;
   font-size: $font-base-size;
   font-weight: 700;
 
   @include large-and-up {
     line-height: $menu-height;
-    margin: 0 20px;
+    margin-right: $sp-2x;
```

Comment:
You probably want to use `margin-inline-end` here instead, so that it works for RTL sites as well